fidouaf/src/main/java/org/ebayopensource/fidouaf/res/FidoUafResource.java

```diff
@@ -87,9 +89,14 @@ public String getStats() {
 	@Path("/public/regRequest/{username}")
 	@Produces(MediaType.APPLICATION_JSON)
 	public RegistrationRequest[] getRegisReqPublic(
-			@PathParam("username") String username) {
+			@PathParam("username") String username, @Context HttpServletRequest httpServletRequest) {

```

Comment:
Let me suggest to use inject of UriInfo instead of HttpServletRequest:

  @Context
  UriInfo uriInfo;